core/src/components/datetime/datetime.ios.scss

```diff
@@ -20,12 +20,16 @@
   @include padding($datetime-ios-padding, $datetime-ios-padding, $datetime-ios-padding, $datetime-ios-padding);
 
   border-bottom: $datetime-ios-border-color;
+  
+  font-size: 14px;
 }
 
 :host .datetime-header .datetime-title {
   color: var(--title-color);
+}
 
-  font-size: 14px;
+:host .datetime-header .datetime-selected-date {
+  @include margin(10px, null, null, null);
 }
```

Comment:
This styling was fairly arbitrary on my part; open to suggestions. I just re-used the same font size and picked a margin that looked good with the existing design.